index.html

```diff
@@ -1,29 +1,250 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css" type="text/css">
     <title>Document</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lora&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap" rel="stylesheet">
+    <script src="https://use.fontawesome.com/d5fa546baf.js"></script>
 </head>
-<body>
- 
-  
-  <footer>
-
-  </footer>
-</body>
-</html>
-
 
+<body>
 
+    <div id="pagemidcont">
+        <!-- the page middel contanier -->
+        <div id="topicscont">
+            <div class="blog1">
+                <div class="imgc"> <img src="images/blog1.png" width="100%" height="100%" alt="blog one image">
```

Comment:
it would be better if you removed the inline style(width, height) from HTML file, and put all the styles on the `style.css` file.

Reply:
okay`